src/decode.rs

```diff
@@ -5135,9 +5135,6 @@ pub fn rav1d_submit_frame(c: &Rav1dContext, state: &mut Rav1dState) -> Rav1dResu
 
     // allocate frame
 
-    // We must take itut_t35 out of the context before the call so borrowck can
-    // see we mutably borrow `c.itut_t35` disjointly from the task thread lock.
-    let itut_t35 = mem::take(&mut state.itut_t35);
```

Comment:
AFAICT this comment was stale, since it compiles just fine to borrow `state.itut_t35` for the following function call rather than `mem::take`ing its value. I also don't see what "the task thread lock" could refer to here.